README.md

```diff
@@ -0,0 +1,61 @@
+# Money Transfer Project
+
+## Prerequisites
+
+Before running this application, ensure you have the following installed:
+
+* [.NET 8.0 SDK](https://dotnet.microsoft.com/en-us/download/dotnet/8.0) or later
+* Temporal Server
```

Comment:
```suggestion
* Temporal CLI
```

The server is a different thing, the CLI happens to embed a dev server and we're using `temporal server start-dev` below